modules/accredited_representative_portal/app/serializers/accredited_representative_portal/power_of_attorney_request_serializer.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class PowerOfAttorneyRequestSerializer
+    include JSONAPI::Serializer
+
+    attributes :id, :claimant_id
+
+    attribute :created_at do |object|
+      object.created_at.iso8601
+    end
+
+    attribute :resolution do |object|
+      next nil if object.resolution.blank?
+
+      AccreditedRepresentativePortal::PowerOfAttorneyRequestResolutionSerializer.new(
+        object.resolution
+      ).serializable_hash[:data][:attributes]
+    end
```

Comment:
Should we use a [relationship serializer](https://github.com/jsonapi-serializer/jsonapi-serializer?tab=readme-ov-file#specifying-a-relationship-serializer)? In fact it looks like we want to follow their example here for polymorphism:
```
class MovieSerializer
  include JSONAPI::Serializer

  has_many :actors, serializer: Proc.new do |record, params|
    if record.comedian?
      ComedianSerializer
    elsif params[:use_drama_serializer]
      DramaSerializer
    else
      ActorSerializer
    end
  end
end
```

Reply:
We can't control the structure that way. The result isn't flat like how we want it to be. And there doesn't seem to be a way to override the default serialization when using relationships.